src/types/tuple.rs

```diff
@@ -566,6 +566,255 @@ macro_rules! tuple_conversion ({$length:expr,$(($refN:ident, $n:tt, $T:ident)),+
         }
     }
 
+    impl<$($T),+> crate::call::private::Sealed for ($($T,)+) {}
```

Comment:
Does this need `where $($T: IntoPyObject<'py>,)` bounds added? I think it cannot hurt.

Reply:
Done